logging/src/main/java/life/qbic/logging/api/LogMessage.java

```diff
@@ -0,0 +1,17 @@
+package life.qbic.logging.api;
+
+/**
+ * Simple logging message record that holds basic information like:
+ * <ul>
+ *   <li>application</li>
+ *   <li>log level</li>
+ *   <li>message</li>
+ *   <li>throwable</li>
+ * </ul>
+ *
+ * @since 1.0.0
+ */
+public record LogMessage(String application, LogLevel logLevel, String message,
+                         Throwable throwable) {
```

Comment:
Just noticed that this is a record. equals and hashCode are implemented by default.
Open question: do we need an id? otherwise the content is used to compare log messages

Reply:
I don't yet see a use case to introduce an ID for LogMessages. If you think that helps mapping logging events to log entries, happy to add one.

Reply:
No just wanted to bring the question up in case they need to be identified by something other than their values. As I do not see a reason for that either, I will resolve this comment.